### ISSUE TITLE
libosrm.pc includes reference to libosmium

### ISSUE THREAD

**Issue by chesty** [Project-OSRM/osrm-backend#2894]
Is it normal for /usr/local/lib/pkgconfig/libosrm.pc to have -I/src/osrm-backend/third_party/libosmium/include in it? ie /src/.. and not /usr/local/include?

I build and install with

```
    mkdir /src && \
    cd /src && \
    git clone --depth 1 https://github.com/Project-OSRM/osrm-backend && \
    cd osrm-backend && \
    mkdir -p build && \
    cd build && \
    cmake .. -DCMAKE_BUILD_TYPE=Release && \
    cmake --build . && \
    cmake --build . --target install
```

I ask because I want to delete /src after installing and I don't know if it will cause a problem with other programs like vroom using libosrm

**Comment by danpat** [comment]
@chesty Hmm, that looks wrong, but harmless.  AFAIK, we don't need the `libosmium` headers to be part of the libosrm.pc file, they're not needed to link against OSRM.  If `/src` goes away, you'll just end up with a harmless `-I` parameter, but we should probably clean it up.

**Comment by danpat** [comment]
@springmeyer looks like you changed this in 6ecc123d.  I'm pretty sure `libosmium` isn't required for downstream consumers of `libosrm`, is this just an oversight?

**Comment by springmeyer** [comment]
> I'm pretty sure libosmium isn't required for downstream consumers of libosrm, is this just an oversight or am I missing something?

Just an oversight. Will take a look at fixing, but this is low priority for me.

**Comment by springmeyer** [comment]
This will be fixed as part of https://github.com/Project-OSRM/osrm-backend/pull/3119 (refs https://github.com/Project-OSRM/osrm-backend/pull/3119/commits/74a989fdfa8b3211292f8162234f85862f62c463)

**Comment by daniel-j-h** [comment]
> This will be fixed as part of #3119 (refs 74a989f)
